merge.py

```diff
@@ -18,6 +18,44 @@
 TMP_DIR_FMT = '/tmp/{dir}'
 
 
+class Command(object):
+    def __init__(self, command, desc, error, shell=False):
+        self.command = command
+        self.desc = desc
+        self.error = error
+        self.shell = shell
+
+
+GIT_CMDS = [
+    Command('git clone -b {branch} {repo_url} .', 'Clone',
```

Comment:
I like it! I'd consider making all but the command a keyword argument (to make it clear what the shorter string is vs. the longer string), but this definitely seems to clean things up!